src/profile/utils.js

```diff
@@ -44,17 +46,39 @@ function getDataItem(data, itemName) {
   }
 }
 
-function displayList(data, userInfoList) {
-  for (const listItem in data) {
-    const innerText = `${formatPropertyField(listItem)}: ${getDataItem(
-      data,
-      listItem,
-    )}`;
+function displayList(primaryData, userInfoList, secondaryData) {
+  const prevSibling = userInfoList.previousSibling.innerHTML;
+  const isOldData = prevSibling.toLowerCase().includes('old');
+  const isNewData = prevSibling.toLowerCase().includes('new');
+
+  for (const listItem in primaryData) {
+    const oldValue = getDataItem(primaryData, listItem);
+    const newValue = getDataItem(secondaryData, listItem);
+    const isValueEqual = String(oldValue).trim() === String(newValue).trim();
+
+    let diffClass;
+    if (!isValueEqual) {
+      diffClass = isOldData ? OLD_DIFF_CLASS : NEW_DIFF_CLASS;
+    }
+
+    const fragment = new DocumentFragment();
+
+    let spanKey = document.createElement('span');
+    spanKey.textContent = `${formatPropertyField(listItem)}: `;
+    fragment.appendChild(spanKey);
+
+    let spanValue = document.createElement('span');
```

Comment:
let's make use of **const** where ever and all possible

Reply:
Thank you for your kind words @Pavangbhat. Can you please check it now?